packages/@lwc/babel-plugin-component/src/__tests__/utils/test-transform.js

```diff
@@ -5,86 +5,20 @@
  * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/MIT
  */
 const babel = require('@babel/core');
-const { stripIndents } = require('common-tags');
-const prettier = require('prettier');
-
+const { LWC_VERSION } = require('@lwc/shared');
+const plugin = require('../../index.js');
 const BASE_CONFIG = { babelrc: false, configFile: false, filename: 'test.js' };
 
-function transform(plugin, pluginOpts = {}, opts = {}) {
-    const testConfig = Object.assign(
-        {},
-        BASE_CONFIG,
-        {
-            plugins: [[plugin, pluginOpts]],
-        },
-        opts
-    );
-
-    return function (source) {
-        return babel.transform(prettify(source), testConfig);
+module.exports = function transform(source, opts = {}) {
```

Comment:
This utility is only used in `packages/@lwc/babel-plugin-component/src/__tests__/index.spec.js`, it can now be inlined.

Reply:
Done.